template/private-npm-package/_package.json

```diff
@@ -21,11 +21,17 @@
   "scripts": {
     "build": "skuba build-package",
     "format": "skuba format",
+    "commit": "cz",
     "lint": "skuba lint",
     "release": "yarn build && skuba release",
     "test": "skuba test --coverage",
     "test:watch": "skuba test --watch"
   },
+  "config": {
```

Comment:
1. I believe we can/should do the same to the `oss-npm-package` template.
2. Should we add a section to the relevant template READMEs?

Reply:
Ok, I was just not sure about the `oss`, just because you could go either way about it and implement `changesets` for example

Reply:
👍 It's definitely easier to get Changesets going in `oss`, but it should be fine to include this out of the box since the template defaults to using Semantic Release anyway.